src/utils/DMRoomMap.js

```diff
@@ -32,6 +41,34 @@ export default class DMRoomMap {
         }
     }
 
+    /**
+     * Returns a shared instance of the class
+     * that uses the singleton matrix client
+     * The shared instance must be started before use.
+     */
+    static shared() {
+        if (!DMRoomMap._sharedInstance) {
+            DMRoomMap._sharedInstance = new DMRoomMap(MatrixClientPeg.get());
```

Comment:
hrm; this will get stuck attached to the old MatrixClient on logout.

How about making start and stop create a new sharedinstance?